### ISSUE TITLE
CookieOverflow error when sending many invoices at once

### ISSUE THREAD

**Issue by rbarreca** [LocalOrbit/localorbit#3493]
**To Reproduce**
1. Go to Financials > Send Invoices for a market that has over 50 uninvoiced invoices
2. Click on checkbox in top-left to select all
3. Hit Send Selected
4. See `ActionDispatch::Cookies::CookieOverflow` error due to `flash` cookie message getting lots of order numbers in it

**Expected behavior**
No error. Would be fine to not show all order numbers and just show `flash` of "x invoices were emailed to buyers. Invoices can be downloaded on the Enter Receipts page. But, there are other bugs related to cookies (i.e. sticky filters) so maybe ActiveRecord session storage is a better fix?

**Screenshots**
<img width="1167" alt="screen shot 2018-11-05 at 10 27 37 pm" src="https://user-images.githubusercontent.com/1458906/48051755-0e630500-e14a-11e8-925f-97632746f2af.png">

**Tech notes**
 - Would a cheap fix be to switch to session storage and run a trim job every X days a la https://stackoverflow.com/a/9474262/444921
- See https://rollbar.com/LocalOrbit/Local-Orbit/items/452/ for instances

**Comment by rbarreca** [comment]
@thermistor any thoughts on switching to https://github.com/rails/activerecord-session_store?

**Comment by thermistor** [comment]
@rbarreca yes, activerecord-session-store is what I would reach for first probably, alternatively a [redis session store](https://github.com/redis-store/redis-rails#session-storage).
